[invalid.py]
  [test_params_context_invalid_type[None\]]
    expected: ERROR
